Synthesis failed for python-os-config

Hello! Autosynth couldn't regenerate python-os-config. :broken_heart:

Here's the output from running `synth.py`:

```
ate, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b241bbc1 feat: Added OAuth scope for the service
2021-01-08 07:18:20,496 autosynth [DEBUG] > Running: git checkout 67f09bf301083798e7220137723222eb00b8f88a
Note: checking out '67f09bf301083798e7220137723222eb00b8f88a'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 67f09bf feat(java): add helper to remove methods in post processing (#883)
2021-01-08 07:18:20,501 autosynth [DEBUG] > Running: git branch -f autosynth-207
2021-01-08 07:18:20,504 autosynth [DEBUG] > Running: git checkout autosynth-207
Switched to branch 'autosynth-207'
2021-01-08 07:18:20,508 autosynth [INFO] > Running synthtool
2021-01-08 07:18:20,508 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-08 07:18:20,508 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/207/sponge_log.log
2021-01-08 07:18:20,510 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-08 07:18:20,723 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-207
nothing to commit, working tree clean
2021-01-08 07:18:20,853 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-08 07:18:20,857 synthtool [DEBUG] > Ensuring dependencies.
2021-01-08 07:18:20,866 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-08 07:18:20,870 synthtool [DEBUG] > Cloning googleapis.
2021-01-08 07:18:20,870 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-08 07:18:20,873 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-08 07:18:23,507 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.16.zip) = 5258bef6f79ae1a869f01ca1b8be1c41053132f1f9032037d339d12ce7ba2f0b
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5258bef6f79ae1a869f01ca1b8be1c41053132f1f9032037d339d12ce7ba2f0b"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:272:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 2.623s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-08 07:18:23,565 autosynth [ERROR] > Synthesis failed
2021-01-08 07:18:23,565 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-08 07:18:23,570 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-08 07:18:23,574 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/008ce5bf-87d6-4df6-a516-9e6d6ecebea4/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
eate a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b241bbc1 feat: Added OAuth scope for the service
2021-01-09 06:27:11,835 autosynth [DEBUG] > Running: git checkout fb53b6fb373b7c3edf4e55f3e8036bc6d73fa483
Note: checking out 'fb53b6fb373b7c3edf4e55f3e8036bc6d73fa483'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at fb53b6f chore(python): skip docfx in main presubmit (#889)
2021-01-09 06:27:11,841 autosynth [DEBUG] > Running: git branch -f autosynth-208
2021-01-09 06:27:11,844 autosynth [DEBUG] > Running: git checkout autosynth-208
Switched to branch 'autosynth-208'
2021-01-09 06:27:11,848 autosynth [INFO] > Running synthtool
2021-01-09 06:27:11,848 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-09 06:27:11,848 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/208/sponge_log.log
2021-01-09 06:27:11,850 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-09 06:27:12,068 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-208
nothing to commit, working tree clean
2021-01-09 06:27:12,205 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-09 06:27:12,211 synthtool [DEBUG] > Ensuring dependencies.
2021-01-09 06:27:12,221 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-09 06:27:12,224 synthtool [DEBUG] > Cloning googleapis.
2021-01-09 06:27:12,225 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-09 06:27:12,228 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-09 06:27:13,935 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.16.zip) = 5258bef6f79ae1a869f01ca1b8be1c41053132f1f9032037d339d12ce7ba2f0b
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5258bef6f79ae1a869f01ca1b8be1c41053132f1f9032037d339d12ce7ba2f0b"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:272:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 1.697s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-09 06:27:13,995 autosynth [ERROR] > Synthesis failed
2021-01-09 06:27:13,995 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-09 06:27:14,000 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-09 06:27:14,004 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/46757253-38e2-4a17-a782-2d1ad3ef2b6a/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 91e206bc chore: update gapic-generator-java to 0.0.18
2021-01-10 06:17:42,615 autosynth [DEBUG] > Running: git checkout fb53b6fb373b7c3edf4e55f3e8036bc6d73fa483
Note: checking out 'fb53b6fb373b7c3edf4e55f3e8036bc6d73fa483'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at fb53b6f chore(python): skip docfx in main presubmit (#889)
2021-01-10 06:17:42,621 autosynth [DEBUG] > Running: git branch -f autosynth-209
2021-01-10 06:17:42,625 autosynth [DEBUG] > Running: git checkout autosynth-209
Switched to branch 'autosynth-209'
2021-01-10 06:17:42,629 autosynth [INFO] > Running synthtool
2021-01-10 06:17:42,630 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-10 06:17:42,630 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/209/sponge_log.log
2021-01-10 06:17:42,631 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-10 06:17:42,852 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-209
nothing to commit, working tree clean
2021-01-10 06:17:42,989 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-10 06:17:42,995 synthtool [DEBUG] > Ensuring dependencies.
2021-01-10 06:17:43,005 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-10 06:17:43,010 synthtool [DEBUG] > Cloning googleapis.
2021-01-10 06:17:43,010 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-10 06:17:43,014 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-10 06:17:44,671 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 1.649s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-10 06:17:44,732 autosynth [ERROR] > Synthesis failed
2021-01-10 06:17:44,733 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-10 06:17:44,738 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-10 06:17:44,742 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/81ccd4c7-38d6-42ea-aa5c-4aba92dc54ae/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 91e206bc chore: update gapic-generator-java to 0.0.18
2021-01-11 06:15:27,019 autosynth [DEBUG] > Running: git checkout fb53b6fb373b7c3edf4e55f3e8036bc6d73fa483
Note: checking out 'fb53b6fb373b7c3edf4e55f3e8036bc6d73fa483'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at fb53b6f chore(python): skip docfx in main presubmit (#889)
2021-01-11 06:15:27,026 autosynth [DEBUG] > Running: git branch -f autosynth-209
2021-01-11 06:15:27,029 autosynth [DEBUG] > Running: git checkout autosynth-209
Switched to branch 'autosynth-209'
2021-01-11 06:15:27,033 autosynth [INFO] > Running synthtool
2021-01-11 06:15:27,033 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-11 06:15:27,033 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/209/sponge_log.log
2021-01-11 06:15:27,035 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-11 06:15:27,255 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-209
nothing to commit, working tree clean
2021-01-11 06:15:27,392 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-11 06:15:27,397 synthtool [DEBUG] > Ensuring dependencies.
2021-01-11 06:15:27,406 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-11 06:15:27,410 synthtool [DEBUG] > Cloning googleapis.
2021-01-11 06:15:27,411 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-11 06:15:27,414 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-11 06:15:29,083 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 1.660s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-11 06:15:29,143 autosynth [ERROR] > Synthesis failed
2021-01-11 06:15:29,143 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-11 06:15:29,148 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-11 06:15:29,152 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/49c720eb-2531-451a-8a92-8887062d5bc8/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at c8bfd324 chore: use name google.cloud.bigquery_storage namespace to avoid conflicts with google.cloud.bigquery
2021-01-12 06:26:42,184 autosynth [DEBUG] > Running: git checkout 16ec872dd898d7de6e1822badfac32484b5d9031
Note: checking out '16ec872dd898d7de6e1822badfac32484b5d9031'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 16ec872 chore: add missing quotation mark (#900)
2021-01-12 06:26:42,190 autosynth [DEBUG] > Running: git branch -f autosynth-212
2021-01-12 06:26:42,194 autosynth [DEBUG] > Running: git checkout autosynth-212
Switched to branch 'autosynth-212'
2021-01-12 06:26:42,198 autosynth [INFO] > Running synthtool
2021-01-12 06:26:42,198 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-12 06:26:42,198 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/212/sponge_log.log
2021-01-12 06:26:42,200 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-12 06:26:42,429 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-212
nothing to commit, working tree clean
2021-01-12 06:26:42,570 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-12 06:26:42,576 synthtool [DEBUG] > Ensuring dependencies.
2021-01-12 06:26:42,586 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-12 06:26:42,590 synthtool [DEBUG] > Cloning googleapis.
2021-01-12 06:26:42,591 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-12 06:26:42,595 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-12 06:26:44,520 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 1.917s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-12 06:26:44,588 autosynth [ERROR] > Synthesis failed
2021-01-12 06:26:44,588 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-12 06:26:44,593 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-12 06:26:44,598 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/562a15b6-0972-4c41-8d0e-6257b515637c/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 46370214 chore: remove microgenerator migration comments in BUILD.bazel files
2021-01-13 06:24:09,119 autosynth [DEBUG] > Running: git checkout 16ec872dd898d7de6e1822badfac32484b5d9031
Note: checking out '16ec872dd898d7de6e1822badfac32484b5d9031'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 16ec872 chore: add missing quotation mark (#900)
2021-01-13 06:24:09,126 autosynth [DEBUG] > Running: git branch -f autosynth-225
2021-01-13 06:24:09,131 autosynth [DEBUG] > Running: git checkout autosynth-225
Switched to branch 'autosynth-225'
2021-01-13 06:24:09,136 autosynth [INFO] > Running synthtool
2021-01-13 06:24:09,136 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-13 06:24:09,136 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/225/sponge_log.log
2021-01-13 06:24:09,138 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-13 06:24:09,374 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-225
nothing to commit, working tree clean
2021-01-13 06:24:09,520 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-13 06:24:09,526 synthtool [DEBUG] > Ensuring dependencies.
2021-01-13 06:24:09,538 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-13 06:24:09,543 synthtool [DEBUG] > Cloning googleapis.
2021-01-13 06:24:09,543 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-13 06:24:09,547 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-13 06:24:12,700 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 3.143s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-13 06:24:12,773 autosynth [ERROR] > Synthesis failed
2021-01-13 06:24:12,773 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-13 06:24:12,779 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-13 06:24:12,784 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8fd3fe8b-7c58-4beb-8f6b-28d17c7a7e1d/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
er checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at add7e03e feat: added V1 API
2021-01-14 14:37:15,747 autosynth [DEBUG] > Running: git checkout 16ec872dd898d7de6e1822badfac32484b5d9031
Note: checking out '16ec872dd898d7de6e1822badfac32484b5d9031'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 16ec872 chore: add missing quotation mark (#900)
2021-01-14 14:37:15,754 autosynth [DEBUG] > Running: git branch -f autosynth-237
2021-01-14 14:37:15,758 autosynth [DEBUG] > Running: git checkout autosynth-237
Switched to branch 'autosynth-237'
2021-01-14 14:37:15,763 autosynth [INFO] > Running synthtool
2021-01-14 14:37:15,763 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-14 14:37:15,763 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/237/sponge_log.log
2021-01-14 14:37:15,765 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-14 14:37:15,998 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-237
nothing to commit, working tree clean
2021-01-14 14:37:16,140 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-14 14:37:16,146 synthtool [DEBUG] > Ensuring dependencies.
2021-01-14 14:37:16,157 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-14 14:37:16,161 synthtool [DEBUG] > Cloning googleapis.
2021-01-14 14:37:16,162 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-14 14:37:16,166 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-14 14:37:17,885 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 1.709s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-14 14:37:17,955 autosynth [ERROR] > Synthesis failed
2021-01-14 14:37:17,955 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-14 14:37:17,960 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-14 14:37:17,965 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5e826317-fa5c-433f-b9a3-f421ae6c6cee/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
 to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 9e209af3 chore: Set namespaces for C#, PHP, and Ruby
2021-01-15 08:22:44,770 autosynth [DEBUG] > Running: git checkout 16ec872dd898d7de6e1822badfac32484b5d9031
Note: checking out '16ec872dd898d7de6e1822badfac32484b5d9031'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 16ec872 chore: add missing quotation mark (#900)
2021-01-15 08:22:44,777 autosynth [DEBUG] > Running: git branch -f autosynth-238
2021-01-15 08:22:44,781 autosynth [DEBUG] > Running: git checkout autosynth-238
Switched to branch 'autosynth-238'
2021-01-15 08:22:44,785 autosynth [INFO] > Running synthtool
2021-01-15 08:22:44,786 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-15 08:22:44,786 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/238/sponge_log.log
2021-01-15 08:22:44,788 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-15 08:22:45,012 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-238
nothing to commit, working tree clean
2021-01-15 08:22:45,151 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-15 08:22:45,156 synthtool [DEBUG] > Ensuring dependencies.
2021-01-15 08:22:45,167 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-15 08:22:45,171 synthtool [DEBUG] > Cloning googleapis.
2021-01-15 08:22:45,171 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-15 08:22:45,175 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-15 08:22:46,839 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 1.655s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-15 08:22:46,916 autosynth [ERROR] > Synthesis failed
2021-01-15 08:22:46,916 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-15 08:22:46,923 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-15 08:22:46,929 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b5e48daa-1759-436b-9fe7-ffce1482b520/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
 checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 61ab0348 feat: Migrate the Bazel rules for the libraries published to google-cloud-ruby to use the gapic-generator-ruby instead of the monolith generator.
2021-01-16 06:32:41,650 autosynth [DEBUG] > Running: git checkout 56ddc68f36b32341e9f22c2c59b4ce6aa3ba635f
Note: checking out '56ddc68f36b32341e9f22c2c59b4ce6aa3ba635f'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 56ddc68 chore: add 3.9 to noxfile template (#909)
2021-01-16 06:32:41,658 autosynth [DEBUG] > Running: git branch -f autosynth-240
2021-01-16 06:32:41,662 autosynth [DEBUG] > Running: git checkout autosynth-240
Switched to branch 'autosynth-240'
2021-01-16 06:32:41,666 autosynth [INFO] > Running synthtool
2021-01-16 06:32:41,667 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-16 06:32:41,667 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/240/sponge_log.log
2021-01-16 06:32:41,669 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-16 06:32:41,908 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-240
nothing to commit, working tree clean
2021-01-16 06:32:42,058 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-16 06:32:42,064 synthtool [DEBUG] > Ensuring dependencies.
2021-01-16 06:32:42,076 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-16 06:32:42,081 synthtool [DEBUG] > Cloning googleapis.
2021-01-16 06:32:42,081 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-16 06:32:42,086 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-16 06:32:44,228 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 2.131s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-16 06:32:44,303 autosynth [ERROR] > Synthesis failed
2021-01-16 06:32:44,303 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-16 06:32:44,310 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-16 06:32:44,315 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/29194dd0-d137-4c19-b14a-efe9aaef350f/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
 checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 61ab0348 feat: Migrate the Bazel rules for the libraries published to google-cloud-ruby to use the gapic-generator-ruby instead of the monolith generator.
2021-01-17 07:53:13,219 autosynth [DEBUG] > Running: git checkout 56ddc68f36b32341e9f22c2c59b4ce6aa3ba635f
Note: checking out '56ddc68f36b32341e9f22c2c59b4ce6aa3ba635f'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 56ddc68 chore: add 3.9 to noxfile template (#909)
2021-01-17 07:53:13,226 autosynth [DEBUG] > Running: git branch -f autosynth-240
2021-01-17 07:53:13,230 autosynth [DEBUG] > Running: git checkout autosynth-240
Switched to branch 'autosynth-240'
2021-01-17 07:53:13,234 autosynth [INFO] > Running synthtool
2021-01-17 07:53:13,234 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-17 07:53:13,235 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/240/sponge_log.log
2021-01-17 07:53:13,236 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-17 07:53:13,455 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-240
nothing to commit, working tree clean
2021-01-17 07:53:13,588 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-17 07:53:13,594 synthtool [DEBUG] > Ensuring dependencies.
2021-01-17 07:53:13,603 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-17 07:53:13,607 synthtool [DEBUG] > Cloning googleapis.
2021-01-17 07:53:13,607 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-17 07:53:13,611 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-17 07:53:15,219 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 1.600s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-17 07:53:15,297 autosynth [ERROR] > Synthesis failed
2021-01-17 07:53:15,298 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-17 07:53:15,303 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-17 07:53:15,307 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0b6bbaa2-1b46-45ec-a4bd-ef2b18c764b2/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
 checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 61ab0348 feat: Migrate the Bazel rules for the libraries published to google-cloud-ruby to use the gapic-generator-ruby instead of the monolith generator.
2021-01-18 06:20:59,988 autosynth [DEBUG] > Running: git checkout 56ddc68f36b32341e9f22c2c59b4ce6aa3ba635f
Note: checking out '56ddc68f36b32341e9f22c2c59b4ce6aa3ba635f'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 56ddc68 chore: add 3.9 to noxfile template (#909)
2021-01-18 06:20:59,995 autosynth [DEBUG] > Running: git branch -f autosynth-240
2021-01-18 06:20:59,999 autosynth [DEBUG] > Running: git checkout autosynth-240
Switched to branch 'autosynth-240'
2021-01-18 06:21:00,004 autosynth [INFO] > Running synthtool
2021-01-18 06:21:00,004 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-18 06:21:00,004 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/240/sponge_log.log
2021-01-18 06:21:00,006 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-18 06:21:00,239 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-240
nothing to commit, working tree clean
2021-01-18 06:21:00,385 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-18 06:21:00,391 synthtool [DEBUG] > Ensuring dependencies.
2021-01-18 06:21:00,402 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-18 06:21:00,406 synthtool [DEBUG] > Cloning googleapis.
2021-01-18 06:21:00,406 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-18 06:21:00,410 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-18 06:21:02,754 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 2.333s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-18 06:21:02,832 autosynth [ERROR] > Synthesis failed
2021-01-18 06:21:02,833 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-18 06:21:02,838 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-18 06:21:02,843 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/fb8e9eb4-93e5-4b75-8cdd-af72ded80dfb/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

Autosynth is still having trouble generating python-os-config. :sob:

Here's the output from running `synth.py`:

```
 checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 61ab0348 feat: Migrate the Bazel rules for the libraries published to google-cloud-ruby to use the gapic-generator-ruby instead of the monolith generator.
2021-01-19 06:12:27,996 autosynth [DEBUG] > Running: git checkout 56ddc68f36b32341e9f22c2c59b4ce6aa3ba635f
Note: checking out '56ddc68f36b32341e9f22c2c59b4ce6aa3ba635f'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 56ddc68 chore: add 3.9 to noxfile template (#909)
2021-01-19 06:12:28,002 autosynth [DEBUG] > Running: git branch -f autosynth-240
2021-01-19 06:12:28,005 autosynth [DEBUG] > Running: git checkout autosynth-240
Switched to branch 'autosynth-240'
2021-01-19 06:12:28,009 autosynth [INFO] > Running synthtool
2021-01-19 06:12:28,010 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-01-19 06:12:28,010 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/python-os-config/240/sponge_log.log
2021-01-19 06:12:28,012 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-01-19 06:12:28,211 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/python-os-config/synth.py.
On branch autosynth-240
nothing to commit, working tree clean
2021-01-19 06:12:28,337 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-19 06:12:28,342 synthtool [DEBUG] > Ensuring dependencies.
2021-01-19 06:12:28,351 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2021-01-19 06:12:28,355 synthtool [DEBUG] > Cloning googleapis.
2021-01-19 06:12:28,356 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-01-19 06:12:28,359 synthtool [DEBUG] > Generating code for: //google/cloud/osconfig/v1/:osconfig-v1-py.
2021-01-19 06:12:29,869 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/cloud/osconfig/v1/:osconfig-v1-py:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-java/archive/v0.0.18.zip) = 0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9
DEBUG: Rule 'gapic_generator_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0d5882ac7c644817b888813369dd81cce3e4df3e0bd79bc4784ef21c8cfa10a9"
DEBUG: Call stack for the definition of repository 'gapic_generator_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:142:1
INFO: SHA256 (https://github.com/bazelbuild/rules_python/archive/0.1.0.tar.gz) = 48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "48f7e716f4098b85296ad93f5a133baf712968c13fbc2fdf3a6136158fe86eac"
DEBUG: Call stack for the definition of repository 'rules_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:42:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.39.1.zip) = 0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0bed83ec9e0f457a8fde09a7e2c2c1ffc6d8f457f88394f9f0994a3939a57836"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:237:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.17.0.tar.gz) = 3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "3ecdb496aa86c684477da31e5a4a8f48f6037c9b22b601627caa5878f571df28"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:271:1
Loading: 0 packages loaded
DEBUG: /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/rules_python/python/pip.bzl:61:5: DEPRECATED: the pip_repositories rule has been replaced with pip_install, please see rules_python 0.1 release notes
ERROR: Skipping '//google/cloud/osconfig/v1/:osconfig-v1-py': The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
WARNING: Target pattern parsing failed.
ERROR: The package part of '//google/cloud/osconfig/v1/:osconfig-v1-py' should not end in a slash
INFO: Elapsed time: 1.501s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-os-config/synth.py", line 34, in <module>
    bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 193, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/cloud/osconfig/v1/:osconfig-v1-py']' returned non-zero exit status 1.
2021-01-19 06:12:29,926 autosynth [ERROR] > Synthesis failed
2021-01-19 06:12:29,926 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7f3b0fa chore: exclude `.nox/` from linting; update python versions in CONTRIBUTING; add pre-commit; update intersphinx links (#32)
2021-01-19 06:12:29,931 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-01-19 06:12:29,935 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/75364393-56ef-4af7-92e3-303ca90d57b3/targets/github%2Fsynthtool;config=default/tests;query=python-os-config;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -31,7 +31,7 @@ for version in versions:
     library = gapic.py_library(
         service="osconfig",
         version=version,
-        bazel_target=f"//google/cloud/osconfig/{version}/:osconfig-{version}-py"
+        bazel_target=f"//google/cloud/osconfig/{version}:osconfig-{version}-py"
     )
 
     s.move(library, excludes=["nox.py", "setup.py", "README.rst", "docs/index.rst"])

